app/grandchallenge/evaluation/views/__init__.py

```diff
@@ -1412,3 +1419,56 @@ def get_context_data(self, *args, **kwargs):
             }
         )
         return context
+
+
+class PhaseStarterKitInfo(
```

Comment:
We usually call information views details views, right? So maybe rename to `PhaseStarterKitDetail`? Then again, it's not a typical detail view, as in we don't have a PhaseStarterKit model.

Reply:
Agreed, I think that this should be a `TemplateView`, that provides the `get_permission_object` returning the phase. The `PhaseStarterKitDownload` is probably just a `View`. No need to go crazy with the mixins for permissions, usually it is a single statement and covered by tests.

Reply:
I agree, 'info' is probably an incorrect name here. Since we don't have a direct model we are viewing or manipulating I think the `DetailView` can indeed far better be replaced by `TemplateView` / `View`.

However, afaik these don't have any permission checks in them. Instead of patching `dispatch()` I found it cleaner to keep using the permission mixin. Not sure: shadowing the `dispatch()` feels wrong for these typical permission checks.

Since these comments also apply to `PhaseArchiveInfo` I've taken the liberty of also applying these changes there.